src/components/PolicyMasterPanel.js

```diff
@@ -303,8 +286,12 @@ class PolicyMasterPanel extends FormPanel {
                   withNull={true}
                   nullLabel={formatMessage(intl, "product", "Product.none")}
                   onChange={this._onProductChange}
-                  filter={this._filterProducts}
                   required={true}
+                  locationId={
+                    !!edited.family
+                      ? decodeId(edited.family.location.parent.parent.parent.id)
```

Comment:
I guess that will get you only the region level product, I think we should have a way in the core to say "location and its parents"

Reply:
No. It will give you district, region and national products.